doc/src/tutorial/basic_operations.rst

```diff
@@ -116,7 +116,15 @@ For example
     x**2 + 3*x - 1/2
     >>> expr.subs(x, 2)
     19/2
-
+    
+Adding two different strings using sympify
```

Comment:
I would rather not emphasise adding strings in the tutorial. The tutorial should suggest `parse_expr` rather than `sympify` in any case.

Reply:
@oscarbenjamin I suggested this because most of the time when we need to add something to the code, it takes up 3-4 lines of code, this way it can be done with just 1 line of code.